terraform/applications/discourse/cloudwatch-logs-to-papertrail.tf

```diff
@@ -0,0 +1,46 @@
+resource "aws_s3_bucket" "logs_to_papertrail" {
+  acl    = "private"
+  bucket = "logs-to-papertrail-lambda-code"
+}
+
+data "aws_caller_identity" "current" {}
+
+resource "aws_lambda_function" "logs_to_papertrail" {
```

Comment:
this doesn't make sense in application-specific environments, since its generically used for shipping cloudwatch lambda logs to papertrail. Moved to accounts terraform since it's scope is an AWS account, and creating a cluster remediation epic ticket on whether or not we even want it.